tests/functional/modules/test_zos_copy_func.py

```diff
@@ -1040,18 +1040,18 @@ def test_copy_dest_directory_remain(ansible_zos_module, src):
                 
         assert permissions_be_copy == permissions_af_copy
     finally:
-        hosts.all.shell(cmd="rm -r {0}".format(dest_path))
+        hosts.all.file(path=dest_path, state="absent")
 
 
 @pytest.mark.uss
 @pytest.mark.parametrize("src", [
-    dict(src="/etc/", is_file=False, is_binary=False, is_remote=False),
-    dict(src="/etc/", is_file=False, is_binary=False, is_remote=True),])
-def test_copy_dest_folder_directory_remain(ansible_zos_module, src):
+    dict(src="/etc/", is_remote=False),
+    dict(src="/etc/", is_remote=True),])
+def test_ensure_copy_folder_dont_change_permission_on_dest(ansible_zos_module, src):
```

Comment:
Same comment from above applies to this case: instead of using `dont` use `does_not`, and I'd use `directory` or `dir` instead of `folder`.